vaccine_feed_ingest/runners/nv/immunizenevada_org/parse.py

```diff
@@ -77,6 +78,33 @@ def extract_locator_data(json_data):
     return result
 
 
+def generate_id(name):
+    """Generate a stable ID for a location from the name.
+
+    We don't want duplicate entries for the same location. If the
+    same location is listed multiple times with minor differences
+    in the name (extra space, or age info in parentheses), we want
+    to produce a single, consistent ID.
+
+    """
+    # Strip off parenthetical age info in the name, like "(16+)".
+    id = re.sub(r"\([0-9+]+\)", "", name)
```

Comment:
Since a lot of this codebase grows by copy-paste... could you add a comment that this might not be safe for other datasets? We've seen some which have titles like `CVS (1234)` which would result in this generating the id of `cvs` :|

Reply:
Good point. As mentioned above, I can make the regex more specific to match only the specific age restriction format:
```
id = re.sub(r"\([0-9]+\+\)", "", name)
```